### PR TITLE
FEAT: Added playlist

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

**Review by Jia-A — APPROVED** [review]
Everything looks good. Your playlist.css is empty, you might want to look into that. Good to merge.

File: src/components/VideoCard/VideoCard.css [hunk 1]
```diff
@@ -1,15 +1,8 @@
 .videos-container {
   display: flex;
-  /* flex-direction: column; */
-  /* justify-content: center; */
-  /* align-items: center; */
-  /* display: grid;
-  grid-template-columns: repeat(3, 1fr); */
+
```
Comment (Megha-Joshi) [inline src/components/VideoCard/VideoCard.css:3]: Remove this commented code, if not required.

File: src/components/VideoCard/VideoCard.jsx [hunk 2]
```diff
@@ -23,17 +28,14 @@ const VideoCard = () => {
...
-  const { videos, history } = videoState;
-
-  // const likeVideoToggleHandler = (video) => {
-  //   const index =
-  //     likedVideos.findIndex((item) => item._id === video._id) === -1;
-  // };
-
-  // }
-  // const isInLikedVideos =
-  //   likedVideos.findIndex((i) => i._id === videos._id) === -1 ? false : true;
```
Comment (Megha-Joshi) [inline src/components/VideoCard/VideoCard.jsx:36]: Remove the commented code.

**Review by Megha-Joshi — COMMENTED** [review]
Everything looks fine. Good to merge

**Pull request merged**